### PR TITLE
Add date format assert

### PR DESCRIPTION
**Pull request by franciscocardoso**
This assert checks if a date is valid in a given format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ The following set of extra asserts are provided by this package:
 - [Date](#date)
 - [DateDiffGreaterThan](#datediffgreaterthan) (requires `moment`)
 - [DateDiffLessThan](#datedifflessthan) (requires `moment`)
+- [DateFormat](#datedifflessthan) (requires `moment`)
```
Comment (ricardogama) [inline README.md]: Bad link.

Comment (franciscocardoso) [inline README.md]: :angel:

**Comment by ruimarinho** [comment]
@franciscocardoso what about using the `Date` assert with `{ format: 'YYYY-MM-DD' }` as an option instead?

**Comment by franciscocardoso** [comment]
I prefer this approach but don't have strong feelings about it.
Do you want me to update to your suggestion?

/cc @ruimarinho

**Comment by ruimarinho** [comment]
Let me ping @nunofgs @fixe to hear their thoughts too.

**Comment by franciscocardoso** [comment]
Deprecated by https://github.com/seegno/validator.js-asserts/pull/95.

**Pull request closed without merging**